### PR TITLE
Account Detail Page

### LINKED ISSUE #90: Account Detail Page
## Story

As a user, I want to view my account balance and transaction history, so I can detect transaction issues and understand my usage patterns.

## Acceptance Criteria

- [ ] User can view their account details
  - [x] NEAR balance
  - [x] Total number of transactions
  - [x] Timestamp of account creation
- [ ] User can view a list of their transactions with details
  - [x] TX type (action name, e.g. Create Account, Sent, Call)
  - [x] Recipient name (e.g. bob.near, crypto-corgis-contract)
  - [x] Amount of NEAR sent (if applicable)
  - [x] TX status (pending, failed, completed)
  - [x] Show pending and failed TXs
  - [x] Show completed TXs
  - [x] Time since status update (e.g. 1 hour)
- [x] Clicking on a TX redirects to the TX's detail page
- [x] URL is /accounts/<address>
- [x] Navigating directly to URL correctly loads the page
- [x] Mobile ok
- [ ] Tested

## Assets

- [Designs](https://projects.invisionapp.com/share/TCS6KG8G9V7#/screens/365131592)

### PR DESCRIPTION
**Pull request by frol**
*Resolves #90*

Here are the current limitations:

* [ ] There is no exposed information about Public key / address and Timestamp of account creation
* [ ] User can view only a list of their *recent* transactions with details (pagination is the topic that is yet to be figured out)
* [ ] Clicking on a TX does not redirect to the TX's detail page since the page is not implemented yet (I think, we should address it once the transaction page is implemented)

### CODE REVIEW

**Comment by frol** [comment]
Here are some accounts:

* https://near-explorer-frontend-pr-93.onrender.com/accounts/studio-2j3m1dxke
* https://near-explorer-frontend-pr-93.onrender.com/accounts/frol.near
* https://near-explorer-frontend-pr-93.onrender.com/accounts/test

File: frontend/components/api/Accounts.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            (SELECT COUNT(transactions.hash) as inTransactionsCount FROM transactions
+              WHERE receiver_id = :id) as inTransactionsCount
+          `,
+          {
+            id
+          }
+        ])
+      ]);
+
+      console.log(rpcAccountResponse, accountResponse);
```
Comment (deacix) [inline frontend/components/api/Accounts.js]: Don't log. If you need logging for testing purposes, check for the right environment or maybe introduce a logger..

**Comment by frol** [comment]
@kcole16 

> Add NEAR symbol after the amount staked

Done 

> Add transaction status next to the timestamp (e.g. "Completed 7 hr ago")

Done (there is no way to discover "non-completed" transactions yet, so I just hardcoded the "Completed" status there)

> Hide the filter and search bar until we have pagination

Done

> Also, what is the Storage column displaying?

It is "used" / "paid" storage. I will send you a draft of the Economics paper in Discord for further reference.

**Comment by kcole16** [comment]
@frol Ah, makes sense re: Storage.

Can we amend the label to "Storage (Used / Paid)" ?

**Comment by frol** [comment]
@kcole16 Done.

**Pull request merged**